types/index.d.ts

```diff
@@ -283,6 +283,16 @@ declare namespace jest {
      */
     toBeSealed(): R;
 
+    /**
+     * Use `.toResolve` when checking if a promise resolves.
+     */
+    toResolve(): R;
```

Comment:
Would you be able to add these types to the `expect` interface too (asymmetric matchers)

Reply:
aha I will have a nosey at the expect now, thanks matt 🎉